docs/root/intro/arch_overview/http/upgrades.rst

```diff
@@ -108,3 +108,18 @@ and `bazel-bin/source/exe/envoy-static --config-path  configs/terminate_connect.
 you will be running two Envoys, the first listening for TCP traffic on port 10000 and encapsulating it in an HTTP/2
 CONNECT request, and the second listening for HTTP/2 on 10001, stripping the CONNECT headers, and forwarding the
 original TCP upstream, in this case to google.com.
+
+Tunneling TCP over HTTP/1.1
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Envoy also has support for tunneling raw TCP over HTTP/1.1 CONNECT requests. This can be used to
+have HTTP clients (or other protocols using TCP as a transport) connecting to
+their destination passing through an HTTP proxy server (e.g. corporate proxy)
+
+[HTTP client] --- raw HTTP --- [Envoy]  --- HTTP tunneled over HTTP/1.1  --- [Upstream HTTP Proxy]  --- raw HTTP  --- [HTTP Server]
+
+Examples of such a set up can be found in the Envoy example config :repo:`directory <configs/>`
+If you run `bazel-bin/source/exe/envoy-static --config-path configs/encapsulate_in_http1_connect.yaml --base-id 1`
+you will be running Envoy listening for TCP traffic on port 10000 and encapsulating it in an HTTP/1.1
+CONNECT request for an HTTP proxy running on localhost and listenig on port
+3128, having as a final destination dest.host.com on port 443.
+passing from .
```

Comment:
yay docs!
is this an extra "passing from ."?

Alternately, do you think we should just have one section

Tunneling TCP over HTTP?

and take out the HTTP/2 specific comments above?

I think we could collapse the two sections, and just call out the two example configs.  WDYT?

Reply:
I'll give a try